examples/sadtalker/README.md

```diff
@@ -0,0 +1,108 @@
+# SadTalker :sob:
+
+A Mindspore implementation of SadTalker based on its [original github](https://github.com/OpenTalker/SadTalker).
+
+## Introduction
+SadTalker is a novel system for a stylized audio-driven single image talking head videos animation using the generated realistic 3D motion coefficients (head pose, expression) of the 3DMM.
+
+![sadtalker_1](https://github.com/hqkate/sadtalker/assets/26082447/f1239f9f-f434-4b2c-8ed0-3f07287eb7f3)
+
+
+## Installation
+
+```bash
+pip install -r requirements.txt
+```
+
+## Data and Preparation
+
+To execute the inference pipeline of SadTalker, please first download the [pretrained checkpoints](#pretrained-checkpoints) and setup the path in [config file](./config/sadtalker.yaml).
+
+### Pretrained checkpoints
+
+You can download the checkpoints from this link. !!!TODO!!!
```

Comment:
better to add links in this PR

Reply:
Well noted, will get it updated once all the checkpoints have been uploaded to the cloud storage and inform you for reviews.

Reply:
Updated, please check when have time, thanks